proxy/src/http/project.rs

```diff
@@ -464,12 +475,24 @@ impl ToDocumentedType for MetadataInput {
     }
 }
 
+/// The domains we support under which a project can live.
```

Comment:
Should also go in `registry.rs`.

Reply:
My reasoning to have it here is that the single purpose of this type it to support the json deserialization of the request, having no other application for the registry.